auth/routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, request, jsonify
+from auth.utils import authenticate, generate_token
+
+auth_bp = Blueprint('auth', __name__)
+
+@auth_bp.route('/login', methods=['POST'])
+def login():
+    data = request.get_json()
+    user = authenticate(data.get('email'), data.get('password'))
+    if user:
+        token = generate_token(user)
+        return jsonify({'token': token}), 200
```

Comment:
data might be null here you need to check
```suggestion
    data = request.get_json()
    if data is None:
        return jsonify({'error': 'Invalid input'}), 400
    user = authenticate(data.get('email'), data.get('password'))
    if user:
        token = generate_token(user)
        return jsonify({'token': token}), 200
```